modules/ROOT/pages/deploy-to-rtf.adoc

```diff
@@ -264,7 +255,10 @@ include::mule-runtime::partial$mmp-concept.adoc[tag=encryptCredentials]
 ----
 +
 NOTE: Make sure that the username and password are not set in the deployment configuration, or they will overwrite the defined server ID.
+=======
+include::mule-runtime::partial$mmp-deploy-to-rtf.adoc[tag=rtfEncryptCredentials]
+>>>>>>> v4.3
```

Comment:
TODO/MERGE CONFLICT: keep changes from v4.3 or v4.4?

Reply:
@Cristian-Venticinque

Reply:
It seems the merge conflict suggests a mixture of includes and content, the whole content of file **deploy-to-rtf.adoc** should be just the following: 

```
= Deploy Applications to Runtime Fabric Using the Mule Maven Plugin

include::mule-runtime::partial$mmp-deploy-to-rtf.adoc[tag=intro]

include::mule-runtime::partial$mmp-deploy-to-rtf.adoc[tag=prerequisites]

include::mule-runtime::partial$mmp-deploy-to-rtf.adoc[tag=rtfDeploymentConfig]

include::mule-runtime::partial$mmp-deploy-to-rtf.adoc[tag=rtfDeploymentCommands]

include::mule-runtime::partial$mmp-deploy-to-rtf.adoc[tag=authenticationMethods]

include::mule-runtime::partial$mmp-deploy-to-rtf.adoc[tag=rtfDeploymentReference]

include::mule-runtime::partial$mmp-deploy-to-rtf.adoc[tag=rtfDeploymentSettingsReference]

include::mule-runtime::partial$mmp-deploy-to-rtf.adoc[tag=rtfEncryptCredentials]

== See Also

* xref:mule-runtime::mmp-concept.adoc[Mule Maven Plugin]
```